db/schema.rb

```diff
@@ -74,6 +74,14 @@
     t.index ["reset_password_token"], name: "index_companies_on_reset_password_token", unique: true
```

Comment:
looks like you have an outdated schema that's ahead of master on your branch. Just delete the schema file locally and push again.